src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -16,9 +16,9 @@ public void init() throws ServletException {
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response)
         throws IOException, ServletException {
-        
+
```

Comment:
Nit: You could undo the whitespace changes here and in ChatServlet.java--that way the history of these files (like you can see [here](https://github.com/Sereezy/CodeU-Spring-2018/commits/master/src/main/java/codeu/controller/ChatServlet.java)) won't show unrelated changes.